data/map systems.txt

```diff
@@ -14319,7 +14345,7 @@ system G-3191
 system G-6183
 	pos 1019.4 -590.18
 	government Uninhabited
-	attributes "blackhole" "umbral reach"
+	attributes "black-hole" "notable-star" "umbral reach"
```

Comment:
This is a fair point. I didn't fuss with the exact spellings while I was borderline hand-combing through the file, because I knew I could easily do a find-and-replace once the legwork was done